src/partials/about.html

```diff
@@ -0,0 +1,19 @@
+<section class="about">
+  <div class="about__head">
+    <p class="about__head-motto">tradition and love</p>
+    <h2 class="about__head-title">How it's made?</h2>
+  </div>
+  <img src="/src/images/about/cow-mobile@1x.png" alt="Zdjęcie krowy z dzbankiem mleka" />
+  <div class="about__description">
```

Comment:
about__desctiption też lepiej zmień na "about__container" lub "about__item" i możesz go edytować poprzez
np:.
about__container:nth-child(2){

}
wedy zmienił bym "about__description-main" na about__description i ja na 12 lini bym zrobił tak:
class="about__description about__desctiption--secondary"

ale twój "about__description-additional" też jest dobry więc decyzję jak to zrobisz zostawiam tobie